src/app/home/home.page.html

```diff
@@ -34,94 +34,92 @@
-                </ion-item>
-              </ion-list>
-            </ion-row>
-            <ion-row class="ion-margin-top">
-              <ion-button *ngIf="meal.additives.length > 0" id="{{meal._key}}-additives" size="small">
-                <ion-icon name="information-circle-outline" slot="start" aria-hidden="true"></ion-icon>
-                <ion-label>Zusatzstoffe</ion-label>
-                <ion-modal trigger="{{meal._key}}-additives" [initialBreakpoint]="1" [breakpoints]="[0, 1]">
-                  <ng-template>
-                    <div class="ion-padding block">
-                      <h2>Zusatzstoffe</h2>
-                      <ion-item lines="none">
-                        <ion-icon name="pizza-outline" slot="start" aria-hidden="true"></ion-icon>
-                        <ion-label class="ion-text-wrap">{{meal.name}}</ion-label>
-                      </ion-item>
-                      <hr />
-                      <ion-list class="list">
-                        <ion-item *ngFor="let additive of meal.additives">
-                          <ion-label class="ion-text-wrap">{{additive.description}}</ion-label>
+  <swiper-container [slidesPerView]="1" [modules]="swiperModules" [virtual]="true" [init]="false" (swiperslidechange)="slideChanged()" #swiper>
+    <swiper-slide *ngFor="let menu of this.selectedCantineData?.menu">
+      <div class="menu-container" id="menu-container">
+        <ion-card *ngFor="let meal of menu.meals" class="meal-card">
+          <ng-template [ngIf]="meal.imageUrl" [ngIfElse]="hideImage">
+            <img class="meal-img" src="{{meal.imageUrl}}" alt="Kein Bild verfügbar" />
+          </ng-template>
+          <ng-template #hideImage>
+            <img class="no-meal-img" alt="Kein Bild verfügbar" />
+          </ng-template>
+          <ion-card-header>
+            <ion-card-title size="small">{{meal.name}}</ion-card-title>
+            <ion-card-subtitle>{{meal.mealCategory}}</ion-card-subtitle>
+          </ion-card-header>
+          <ion-card-content>
+            <ion-grid>
+              <ion-row>
+                <ion-list lines="none">
+                  <ion-item class="ion-no-padding">
+                    <ion-icon name="pricetag-outline" slot="start" aria-hidden="true"></ion-icon>
+                    <ion-label> Studenten: {{meal.studentPrice | number:'.2-2':'de-DE'}}€ / Gast: {{meal.normalPrice | number:'.2-2':'de-DE'}}€</ion-label>
+                  </ion-item>
+                  <ion-item class="ion-no-padding">
+                    <ion-icon name="leaf-outline" slot="start" aria-hidden="true" color="{{meal.co2ClassInfo}}"></ion-icon>
+                    <ion-label>CO<sub>2</sub>/Portion: {{meal.co2PerPortion}}g</ion-label>
+                  </ion-item>
+                </ion-list>
+              </ion-row>
+              <ion-row class="ion-margin-top">
+                <ion-button *ngIf="meal.additives.length > 0" id="{{meal._key}}-additives" size="small">
```

Comment:
Die id ist nicht mehr eindeutig, da jetzt alle Speisepläne der ausgewählten Kantine in die HTML eingebunden werden.
Lösung: id zu "{{meal._key}}-{{menu.date}}-additives" ändern.